packages/elpi/elpi.1.14.2/opam

```diff
@@ -0,0 +1,86 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Claudio Sacerdoti Coen" "Enrico Tassi" ]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/LPCIC/elpi"
+doc: "https://LPCIC.github.io/elpi/"
+dev-repo: "git+https://github.com/LPCIC/elpi.git"
+bug-reports: "https://github.com/LPCIC/elpi/issues"
+
+build: [
+  ["dune" "subst"] {dev}
+  [make "build" "DUNE_OPTS=-p %{name}% -j %{jobs}%"]
+  [make "tests" "DUNE_OPTS=-p %{name}%" "TIMEOUT=240"] {with-test & os != "macos" & os-distribution != "alpine" & os-distribution != "freebsd"}
```

Comment:
Is there any way the time constrained tests could be disabled as well? It always fails somewhere for a reason or another (today was on the s390x machine) and generally doesn’t play well with automated systems where the specs of the machines can be random.
```suggestion
  [make "tests" "DUNE_OPTS=-p %{name}%"] {with-test & os != "macos" & os-distribution != "alpine" & os-distribution != "freebsd"}
```

Reply:
It is surely implementable on my side, since the perf tests are tagged as such. but would require another release I'm afraid.

Reply:
I'm giving it a shot right now...